packages/salesforcedx-vscode-apex/test/jest/commands/metadataOrchestrator.test.ts

```diff
@@ -58,7 +58,7 @@ describe('MetadataOrchestrator', () => {
       ];
       jest.spyOn(orchestrator, 'validateEligibility').mockResolvedValue(mockResponse);
       await expect(orchestrator.validateMetadata(editorStub.document.uri, true)).rejects.toThrow(
-        'Method someMethod is not eligible for OpenAPI Document creation. It is not annotated with @AuraEnabled or has wrong access modifiers.'
+        'Method someMethod is not eligible for OpenAPI Document creation. It is not annotated with an http annotator or has wrong access modifiers.'
```

Comment:
@mingxuanzhangsfdx Thanks for catching the AuraEnabled message! I believe this message should be more general. Http annotations are the current ones, but I imagine AuraEnabled will eventually return. What do think about referring the user to current documentation regarding what makes an Apex method eligible.